openlane/steps/odb.py

```diff
@@ -278,6 +278,16 @@ class ReportDisconnectedPins(OdbpyStep):
     id = "Odb.ReportDisconnectedPins"
     name = "Report Disconnected Pins"
 
+    config_vars = OdbpyStep.config_vars + [
+        # Why is this a PDK variable again
```

Comment:
Because it references PDK cells so it is PDK dependent.

Reply:
Fair enough